app/src/main/java/org/kiwix/kiwixmobile/KiwixWebViewClient.java

```diff
@@ -2,12 +2,15 @@
 
 import android.content.Intent;
 import android.net.Uri;
+import android.support.design.widget.FloatingActionButton;
 import android.util.Log;
 import android.view.LayoutInflater;
+import android.view.View;
 import android.webkit.MimeTypeMap;
 import android.webkit.WebView;
 import android.webkit.WebViewClient;
 import android.widget.LinearLayout;
```

Comment:
Is there any use of this import now ? Please make sure you optimize imports before pushing :smile: .